linera-core/src/client.rs

```diff
@@ -299,6 +315,10 @@ impl<P, S> ChainClient<P, S> {
     pub fn pending_block(&self) -> &Option<Block> {
         &self.pending_block
     }
+
+    pub fn pending_blobs(&self) -> &BTreeMap<BlobId, HashedBlob> {
+        &self.pending_blobs
+    }
```

Comment:
I find the getters not great. An additional problem is that the `pending_blobs` is public.
So, either remove the `pub` or remove the getter.